pandas/core/arrays/period.py

```diff
@@ -253,6 +253,10 @@ def _from_sequence(
         ordinals = libperiod.extract_ordinals(periods, freq)
         return cls(ordinals, freq=freq)
 
+    @classmethod
+    def _from_sequence_of_strings(cls, strings, dtype=None, copy=False):
+        return cls._from_sequence(strings, dtype, copy)
```

Comment:
Why does this need a new method?

Reply:
This is the method that gets used by the parser when reading Extension Arrays. The issue explains it with a bit more context: https://github.com/pandas-dev/pandas/issues/26934